ui/src/components/profile/ProfileContainer.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box, Container } from '@mui/material';
+import { FC, ReactElement } from 'react';
+import ProfileDesktopView from './ProfileDesktopView';
+import ProfileMobileView from './ProfileMobileView';
+
+const ProfileContainer: FC = (): ReactElement => {
+  return (
+    <Box>
```

Comment:
actually this is good, but i think it's better to use `useMediaQuery` like randall suggested. you can check out my files or randall's files on how to implement.

it's cleaner that way without the zIndex, etc